src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimSpecTest.java

```diff
@@ -0,0 +1,585 @@
+            ExternalTask paymentTask = assertNextExternalTask(PROCESS_PAYMENT_TOPIC);
+            assertCompleteExternalTask(
+                paymentTask,
+                PROCESS_PAYMENT_TOPIC,
+                MAKE_PBA_PAYMENT_EVENT,
+                MAKE_PAYMENT_ACTIVITY_ID,
+                variables
+            );
+
+            //complete the document generation
+            variables.putValue(
+                FLOW_STATE,
+                FlowState.PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_ONE_V_ONE_SPEC.fullName()
+            );
+            ExternalTask documentGeneration = assertNextExternalTask(PROCESS_CASE_EVENT);
+            assertCompleteExternalTask(
+                documentGeneration,
+                PROCESS_CASE_EVENT,
+                GENERATE_CLAIM_FORM_EVENT,
+                GENERATE_CLAIM_FORM_ACTIVITY_ID,
+                variables
+            );
+
+            //complete the claim issue
+            ExternalTask claimIssue = assertNextExternalTask(PROCESS_CASE_EVENT);
+            assertCompleteExternalTask(
+                claimIssue,
+                PROCESS_CASE_EVENT,
+                PROCESS_CLAIM_ISSUE_EVENT,
+                PROCESS_CLAIM_ISSUE_UNREPRESENTED_ACTIVITY_ID,
+                variables
+            );
+
+            //complete the notification
+            ExternalTask notificationTask = assertNextExternalTask(PROCESS_CASE_EVENT);
+            assertCompleteExternalTask(
+                notificationTask,
+                PROCESS_CASE_EVENT,
+                NOTIFY_APPLICANT_SOLICITOR1_ONLINE_ISSUE_EVENT,
+                NOTIFY_APPLICANT_SOLICITOR1_ONLINE_UNREPRESENTED_ACTIVITY_ID
+            );
+
+            //complete the respondent notification
+            ExternalTask notificationRespondentTask = assertNextExternalTask(PROCESS_CASE_EVENT);
+            assertCompleteExternalTask(
+                notificationRespondentTask,
+                PROCESS_CASE_EVENT,
+                NOTIFY_RESPONDENT1_FOR_CLAIM_CONTINUING_ONLINE_SPEC_EVENT,
+                NOTIFY_RESPONDENT1_FOR_CLAIM_CONTINUING_ONLINE_SPEC_ACTIVITY_ID
+            );
+
+            //complete the Robotics notification
+            ExternalTask forRobotics = assertNextExternalTask(PROCESS_CASE_EVENT);
+            assertCompleteExternalTask(
+                forRobotics,
+                PROCESS_CASE_EVENT,
+                NOTIFY_RPA_ON_CONTINUOUS_FEED_EVENT,
+                NOTIFY_RPA_ON_CONTINUOUS_FEED_ACTIVITY_ID,
+                variables
+            );
+
+            //end business process
+            ExternalTask endBusinessProcess = assertNextExternalTask(END_BUSINESS_PROCESS);
+            completeBusinessProcess(endBusinessProcess);
+
+            assertNoExternalTasksLeft();
+        }
+
+        @Test
+        void shouldSuccessfullyCompleteCreateClaim_whenClaimOfflineMultipartyForUnrepresentedDefendant() {
+            //assert process has started
+            assertFalse(processInstance.isEnded());
```

Comment:
seems like we are duplicating code here. and having to add lots of comments explaining what is going on. wouldn't it be better to create create reusable methods here that would also explain what is going on? i am not a fan of comments like this as well to be honest. i think methods are better in this case. just my opinion. For example we could create a method called givenPorocessHasStarted or something like that. what do you think?